### PR TITLE
Fixed mount permissions with new iso

### PR DESCRIPTION
**Pull request by aaron-prindle**
ref #1003 
includes changes from: #1030

### CODE REVIEW

File: deploy/iso/minikube-iso/package/automount/minikube-automount [hunk 1]
```diff
@@ -163,3 +163,44 @@ fi
 
 # /etc dirs are initialised from /usr/local, to allow the user/admin to customise
 mkdir -p /var/lib/boot2docker/etc/
+
+# VirtualBox Guest Additions
+# - this will bail quickly and gracefully if we're not in VBox
+if modprobe vboxguest &> /dev/null && modprobe vboxsf &> /dev/null; then
+	# fire up VBoxService to do timesync, etc
+	# VBoxService --disable-automount
```
Comment (dlorenc) [inline deploy/iso/minikube-iso/package/automount/minikube-automount]: Could you just delete this?

File: deploy/iso/minikube-iso/package/automount/minikube-automount [hunk 2]
```diff
@@ -163,3 +163,44 @@ fi
 
 # /etc dirs are initialised from /usr/local, to allow the user/admin to customise
 mkdir -p /var/lib/boot2docker/etc/
+
+# VirtualBox Guest Additions
```
Comment (dlorenc) [inline deploy/iso/minikube-iso/package/automount/minikube-automount]: Could you add some comments saying where this came from?

Comment (aaron-prindle) [inline deploy/iso/minikube-iso/package/automount/minikube-automount]: Done

File: deploy/iso/minikube-iso/package/automount/minikube-automount (hunk 1 above)
Comment (aaron-prindle) [inline deploy/iso/minikube-iso/package/automount/minikube-automount]: Done

**Pull request merged**